internal/checks/checks.go

```diff
@@ -53,7 +53,7 @@ func (e FatalError) Error() string { return string(e) }
 var _ error = FatalError("")
 
 const (
-	errCapabilityK6Missing = FatalError("k6 is required for scripted check support")
+	errCapabilityK6Missing = FatalError("k6 is required for scripted check support - configure k6 or edit probe capabilities in the SM app")
```

Comment:
```suggestion
	errCapabilityK6Missing = FatalError("k6 is required for scripted check support - configure k6 or disable the probe's k6 capability in the SM app")
```

Thoughts on this rewording? I can't find a good succinct way to describe this.

Reply:
What comes to my mind is
```suggestion
	errCapabilityK6Missing = FatalError("synthetic monitoring backend requested k6 to be enabled on this probe, but it is disabled by CLI arg")
```